test/test11.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+export LD_LIBRARY_PATH=build/
+if build/marc --gobbletygook output=test.bin; then
```

Comment:
Here the test seems to be for invalid argument and inaccessible input file?. Can you split these two tests? If that is not the intention, provide a valid input file.